libraries/cyclestreets-view/build.gradle

```diff
@@ -6,7 +6,8 @@ dependencies {
   compile 'com.getpebble:pebblekit:3.1.0@aar'
 
   // In the main app project, we're already pulling in 23.x via the `acra` dependency.
-  compile 'com.android.support:appcompat-v7:23.4.0'
-  compile 'com.android.support:support-v4:23.4.0'
-  compile 'com.android.support:support-annotations:23.4.0'
+  compile 'com.android.support:appcompat-v7:27.0.2'
+  compile 'com.android.support:support-v4:27.0.2'
+  compile 'com.android.support:support-annotations:27.0.2'
```

Comment:
It concerns me that we're pulling in `com.android.support:support-annotations:27.0.2` here, and `com.android.support:support-annotations:25.3.1` in the main app project.  Can we get these aligned (or perhaps remove the latter dependency entirely?)

Also, is the comment above these dependencies perhaps no longer relevant?

Reply:
I addressed this also in https://github.com/jezhiggins/cyclestreets-android/pull/48.